optout/src/main/java/gov/cms/ab2d/optout/OptOutProcessorImpl.java

```diff
@@ -27,16 +27,23 @@
     @Override
     @Transactional
     public void process() {
-        log.info("import opt-out data - start ...");
 
-        try (var inputStreamReader = s3Gateway.getOptOutFile();
-             var bufferedReader = new BufferedReader(inputStreamReader)
-        ) {
+        final List<String> filenames = s3Gateway.listOptOutFiles();
+        filenames.stream().forEach(fileName -> fetchOptOutFile(fileName));
+
+    }
+
+    private void fetchOptOutFile(final String filename) {
```

Comment:
Since this method does more than fetching files, we should probably rename to `fetchAndProcessOptOutFile`

Reply:
done